bindata/network/openshift-sdn/rules.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+  prometheus: k8s
+  role: alert-rules
+  name: networking-rules
+  namespace: openshift-sdn
+spec:
+  groups:
+  - name: general.rules
+    rules: 
+    - alert: NodeWithoutOVSPod
+      annotations:
+        message: |
+          All nodes should be running an ovs pod. One or more are not 
+      expr: |
+        :kube_pod_info_node_count: != sum(kube_pod_container_info{namespace="openshift-sdn", pod=~"ovs.*"}) 
```

Comment:
OK, so I don't know a lot about prometheus. Will this trigger immediately upon that condition changing? We don't want to have it generating "alerts" during an upgrade (when each node is expected to briefly not be running OVS while it restarts) or when a new node is added to the cluster (in which case the "node count" will presumably go up before the OVS pod gets a chance to start).

Reply:
It would trigger immediately, what kind of a delay would be appropriate?

Reply:
Hm... I'm not sure this check makes sense at all actually. If the pods are *crashing* then that's alert-worthy, but that should be caught by the other check. If they are not running for some reason other than "because they crashed" then presumably it's intentional.